src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -232,7 +232,7 @@ public static string FormatScript(string script)
             return script;
         }
 
-        private static string GetName(string name)
+        public static string GetName(string name)
```

Comment:
Is the robust enough to handle databases with brackets in the name? I made the same comment in another PR.

Reply:
not using this anymore